code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -410,7 +410,7 @@
 	desc = "Particularly twisted deities grant gifts of dubious value."
 	icon_state = "arm_blade"
 	item_state = "arm_blade"
-	flags = ABSTRACT | NODROP
+	flags = ABSTRACT | NODROP | FORCE_VISIBLE
 	w_class = WEIGHT_CLASS_HUGE
```

Comment:
Can you just remove the ABSTRACT flag on these? Basically all they do is hide it from examine, having it stay in the hand is handled by NODROP. I don't really think theres any reason for FORCE_VISIBLE